tests/test_main.py

```diff
@@ -372,10 +373,14 @@ def test_not_render_rules_also_matched(z9324d_extractor, capsys):
 def test_backend_option(capsys):
     if sys.version_info > (3, 0):
         path = get_data_path_by_name("pma16-01")
-        assert capa.main.main([path, "-b", capa.main.BACKEND_VIV]) == 0
+        assert capa.main.main([path, "-j", "-b", capa.main.BACKEND_VIV]) == 0
         std = capsys.readouterr()
-        assert "check for PEB NtGlobalFlag flag (24 matches)" in std.out
+        std_json = json.loads(std.out)
```

Comment:
I am not sure how expensive it is to convert the output to json. Maybe it is enough to check if the strings `VivisectFeatureExtractor` and `matches` are in the output 🤔 @williballenthin  @mr-tz any other ideas?

Reply:
all render modes should be equally "expensive" and I like your approach here

Reply:
> I am not sure how expensive it is to convert the output to json

With this I meant the call to `json.loads` (not the `-j` option) as we are reading the ouput and then parsing it to JSON with `json.loads`.